sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -877,9 +877,10 @@ def collect_index_usage_metrics(self):
         if not self._index_usage_last_check_ts or now - self._index_usage_last_check_ts > interval:
             self._index_usage_last_check_ts = now
             self.log.debug('Collecting index usage statistics')
-            db_names = [d.name for d in self.databases] or [
-                self.instance.get('database', self.connection.DEFAULT_DATABASE)
-            ]
+            default_database = self.instance.get('database', self.connection.DEFAULT_DATABASE)
+            # Filter out tempdb as the query might be blocking and it's index usage information is not relevant
+            db_names = [d.name for d in self.databases if d != 'tempdb'] or ([default_database] if default_database != 'tempdb' else [])
```

Comment:
We've been gathering fragmentation data for `tempdb` so far. Meanwhile, we've realized that this approach doesn't make sense due to the transient nature of objects in `tempdb`. However, we should keep in mind that we're modifying the current behaviour. Consequently, we should allow the possibility of edge cases where certain customers might still find this information beneficial. Although the probability of this scenario is low, I recommend controlling the behavior with a configuration parameter instead of hardcoding `tempdb`. By default, data collection for `tempdb` should be disabled.

Reply:
thx, for the comment, will implemet.